post_introspect.yml

```diff
@@ -25,8 +25,7 @@
             openstack baremetal introspection data save {{ item }} | jq . | grep by-path | cut -d \" -f 4
           register: storage_disks_info
           with_items: "{{ total_nodes.stdout_lines | default([]) }}"
-          when: composable_roles != true
-
+          when: composable_roles != true and osp_release|int < 17
```

Comment:
I don't think 'osp_release|int < 17' is required here. because this task is defined under a block, which already satisfies this condition[1].
[1]

Reply:
yea, right. This isn't needed.